<launch>
  <param name="/camera/driver/depth_registration" value="False" />
  
  <node pkg="rbx1_apps" name="object_follower" type="object_follower.py" output="screen">
  
    <remap from="camera_info" to="/camera/rgb/camera_info" />
    <remap from="depth_image" to="/camera/depth/image_rect" />

    <rosparam>
       rate: 10
       max_z: 2.0
       goal_z: 0.75
       z_threshold: 0.05
       x_threshold: 0.1
       z_scale: 0.5
       x_scale: 2.0
       max_rotation_speed: 2.0
       min_rotation_speed: 0.2
       max_linear_speed: 0.2
       min_linear_speed: 0.02
    </rosparam>
    
  </node>
</launch>
